apps/templates/password_change/email.html

```diff
@@ -0,0 +1,5 @@
+{% block container %}
+    <p>
+        Your password has been changed sucessfully.
```

Comment:
@thenav56  I've replied here.
```suggestion
        Your password has been changed successfully.
```